modules/imgimport/src/imgimport.cpp

```diff
@@ -29,17 +29,46 @@
     SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
 */
 
+#include <opencv2/highgui/highgui.hpp>
+#include <opencv2/core/core.hpp>
+#include <iostream>
+#include "frame.h"
+#include "frame.cpp"
 #include "imgimport.h"
 #include <vector>
+#include <string>
+#include <dirent.h>
 
-ImageImport::ImageImport(std::string telemetry_path, std::string filePath, std::vector<int> videoDeviceNums) {
+using namespace cv;
+using namespace std;
 
+ImageImport::ImageImport(std::string telemetry_path, std::string filePath, std::vector<int> videoDeviceNums) {
+    this->videoDeviceNums=videoDeviceNums;
+    mdls=readcsv(telemetry_path.c_str());
+    dr=opendir(filePath.c_str());
 }
 
 ImageImport::~ImageImport(){
-
+    cout<<"image import ends."<<endl;
```

Comment:
You should use BOOST_LOG_TRIVIAL (or some other variant of boost log) that way we can filter log messages globally.